docs/source/guide/guide_05-factories.rst

```diff
@@ -19,39 +19,39 @@ The typical tasks of a factory are:
 
 2. Determine the noise scale factor at which the next computation should be run;
 
-3. Given the history of noise scale factors and results 
+3. Given the history of noise scale factors and results
    (respectively stored in the object attributes ``self.instack``
    and ``self.outstack``), evaluate the associated zero-noise extrapolation.
 
 The structure of the :class:`.Factory` class is adaptive by construction, since the choice of the next noise
 level can depend on the history of ``self.instack`` and ``self.outstack``. Obviously, non-adaptive
 methods are supported too and they actually represent the most common choice.
 
-Specific classes derived from the abstract class :class:`.Factory`, like :class:`.LinearFactory`, 
+Specific classes derived from the abstract class :class:`.Factory`, like :class:`.LinearFactory`,
 :class:`.RichardsonFactory`, etc., represent different zero-noise extrapolation methods.
-All the built-in factories can be found in the module :py:mod:`mitiq.factories` and
+All the built-in factories can be found in the module :py:mod:`mitiq.zne.inference` and
 are summarized in the following table.
 
 .. _built-in-factories:
 
    .. autosummary::
       :nosignatures:
 
-      mitiq.factories.LinearFactory
-      mitiq.factories.RichardsonFactory
-      mitiq.factories.PolyFactory
-      mitiq.factories.ExpFactory
-      mitiq.factories.PolyExpFactory
-      mitiq.factories.AdaExpFactory
+      mitiq.zne.inference.LinearFactory
+      mitiq.zne.inference.RichardsonFactory
+      mitiq.zne.inference.PolyFactory
+      mitiq.zne.inference.ExpFactory
+      mitiq.zne.inference.PolyExpFactory
+      mitiq.zne.inference.AdaExpFactory
 
 
-Once instantiated, a factory can be passed as an argument to the high-level functions contained in the module :py:mod:`mitiq.zne`.
-Alternatively, a factory can be directly used to implement a zero-noise extrapolation procedure in a fully self-contained way. 
+Once instantiated, a factory can be passed as an argument to the high-level functions contained in the module :py:mod:`mitiq.zne.zne`.
+Alternatively, a factory can be directly used to implement a zero-noise extrapolation procedure in a fully self-contained way.
 
 To clarify this aspect, we now perform the same zero-noise extrapolation with both methods.
 
 ===============================================================
-Using a factory object with the :py:mod:`mitiq.zne` module
+Using a factory object with the :py:mod:`mitiq.zne.zne` module
```

Comment:
```suggestion
Using a factory object with the :py:mod:`mitiq.zne` module
```

Reply:
@nathanshammah `mitiq.zne` is the module no?

Reply:
mitiq.zne is the package.